### PR TITLE
Add useSelector and useDispatch example

### PR DESCRIPTION
**Pull request by ivankoleda**
## Description
This  PR adds an example of usage `useSelector` and `useDispatch`. Mostly based on `react-redux` docs https://react-redux.js.org/using-react-redux/static-typing.

Hesitated about adding this types hooks in hooks folder, but there are examples of *usage* of react only hooks, so I added them into connected folder.

## Related issues:
- Related to #196 and partially resolves it. 

## Checklist

* [x] I have read [CONTRIBUTING.md](https://github.com/piotrwitek/react-redux-typescript-guide/blob/master/CONTRIBUTING.md)
* [x] I have edited `README_SOURCE.md` (NOT `README.md`)
* [x] I have run CI script locally `npm run ci-check` to generate an updated `README.md`
* [x] I have linked all related issues above
* [x] I have rebased my branch

### CODE REVIEW

**Comment by ivankoleda** [comment]
I may add examples of usage of `useRouteMatch` and `useParams` from `react-router` v5 in this PR  to completely resolve this issue. But I am not quite sure where to put them.

File: playground/src/connected/fc-counter-connected-hooks.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Dispatch } from 'redux';
```
Comment (piotrwitek) [inline playground/src/connected/fc-counter-connected-hooks.ts]: I would suggest to put this file in the `/store` folder and call it `hooks.ts`.

File: playground/src/connected/fc-counter-connected-hooks.usage.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```
Comment (piotrwitek) [inline playground/src/connected/fc-counter-connected-hooks.usage.tsx]: Please move this file to `/hooks` folder, and call it `react-redux-hooks.tsx`,

**Comment by piotrwitek** [comment]
Looks good, just need to improve file organization. Thanks!

**Review by piotrwitek — APPROVED** [review]

**Comment by piotrwitek** [comment]
Looks good

**Pull request merged**